### PR TITLE
Change docs for mqtt-kafka config

### PR DESCRIPTION
**Pull request by bmaidics**

### CODE REVIEW

File: src/reference/config/bindings/binding-mqtt-kafka.md [hunk 1]
```diff
@@ -102,6 +129,117 @@ Kafka topic used for routing mqtt messages.
...
+    with:
+      messages: mqtt-devices
+    exit: kafka_cache_client
+```
+
+### routes[].guarded
+
+> `object` as named map of `string:string` `array`
+
+Roles required by named guard.
```
Comment (vordimous) [inline src/reference/config/bindings/binding-mqtt-kafka.md:175]: It would be better to use the same text as the other `guarded` sections unless this works differently

```
List of roles required by each named guard to authorize this route.
```

**Review by vordimous — CHANGES_REQUESTED** [review]

File: src/reference/config/bindings/binding-mqtt-kafka.md [hunk 2]
```diff
@@ -21,7 +21,25 @@ mqtt_kafka_proxy:
...
+  routes:
+    - when:
+        - publish:
+            - topic: /clients/#
+        - subscribe:
+            - topic: /clients/#
+      with:
+        messages: mqtt-clients
+      exit: kafka_cache_client
+    - when:
```
[the remark this answers is not in the document]
Comment (jfallows) [reply src/reference/config/bindings/binding-mqtt-kafka.md:35]: Yes, but dependent on corresponding changes coming on `feature/mqtt-kafka` branch of `zilla`.

File: src/reference/config/bindings/binding-mqtt-kafka.md [hunk 3]
```diff
@@ -102,6 +130,123 @@ Kafka topic used for routing mqtt messages.
...
+          - topic: /clients/#
+      - subscribe:
+          - topic: /clients/#
+```
+
+#### when[].publish[]
+
+> `array`
+
+Array of MQTT topics.
```
Comment (jfallows) [inline src/reference/config/bindings/binding-mqtt-kafka.md:203]:
```suggestion
Array of `mqtt` topic filters matching topic names for publish.
```

File: src/reference/config/bindings/binding-mqtt-kafka.md [hunk 4]
```diff
@@ -102,6 +130,123 @@ Kafka topic used for routing mqtt messages.
...
+
+> `string`
+
+Mqtt topic name.
+
+#### when[].subscribe[]
+
+> `array`
+
+Array of MQTT topics.
```
Comment (jfallows) [inline src/reference/config/bindings/binding-mqtt-kafka.md:220]:
```suggestion
Array of `mqtt` topic filters matching topic names for subscribe.
```

File: src/reference/config/bindings/binding-mqtt-kafka.md [hunk 5]
```diff
@@ -102,6 +130,123 @@ Kafka topic used for routing mqtt messages.
...
+```yaml
+- publish:
+    - topic: /clients/#
+```
+
+##### publish[].topic
+
+> `string`
+
+Mqtt topic name.
```
Comment (jfallows) [inline src/reference/config/bindings/binding-mqtt-kafka.md:214]:
```suggestion
`mqtt` topic filter pattern.
```

File: src/reference/config/bindings/binding-mqtt-kafka.md [hunk 6]
```diff
@@ -102,6 +130,123 @@ Kafka topic used for routing mqtt messages.
...
+```yaml
+- subscribe:
+    - topic: /clients/#
+```
+
+##### subscribe[].topic
+
+> `string`
+
+Mqtt topic filter.
```
Comment (jfallows) [inline src/reference/config/bindings/binding-mqtt-kafka.md:231]:
```suggestion
`mqtt` topic filter pattern.
```

File: src/how-tos/install.md [hunk 7]
```diff
@@ -38,7 +38,7 @@ docker run -v $(pwd)/zilla.yaml:/etc/zilla/zilla.yaml ghcr.io/aklivity/zilla:lat
 
 ## Running Zilla via Helm
 
-Go to the [Zilla artifacthub](https://artifacthub.io/) page to find out more on how to install Zilla using Helm
+Go to the [Zilla artifacthub](https://artifacthub.io/packages/helm/zilla/zilla) page to find out more on how to install Zilla using Helm
```
Comment (jfallows) [inline src/how-tos/install.md:41]: Is this change intentional?

File: .markdownlint.yaml [hunk 8]
```diff
@@ -1,13 +1,10 @@
 default: true
 
 # markdownlint relaxed rules https://github.com/DavidAnson/markdownlint/blob/main/style/relaxed.json
-whitespace: false
 line-length: false
 ul-start-left: false
 ul-indent: false
 no-inline-html: false
-no-bare-urls: false
-fenced-code-language: false
```
Comment (jfallows) [inline .markdownlint.yaml:10]: Are these changes intentional?

File: .github/workflows/build-deploy-docs.yml [hunk 9]
```diff
@@ -43,7 +43,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 18
+          node-version: 20
```
Comment (jfallows) [inline .github/workflows/build-deploy-docs.yml:46]: Do we need to upgrade node for these doc changes?

**Review by jfallows — CHANGES_REQUESTED** [review]

File: src/reference/config/bindings/binding-mqtt-kafka.md [hunk 10]
```diff
@@ -102,6 +130,123 @@ Kafka topic used for routing mqtt messages.
 
 Compacted Kafka topic for storing mqtt retained messages.
 
+#### options.clients
+
+> `string`
```
[the remark this answers is not in the document]
Comment (bmaidics) [reply src/reference/config/bindings/binding-mqtt-kafka.md:135]: Yep, you're right

File: src/how-tos/install.md (hunk 7 above)
Comment (bmaidics) [reply src/how-tos/install.md:41]: I have no idea how these were changed by my PR. Fixed

Comment (vordimous) [reply src/how-tos/install.md:41]: Yes, I added them.

**Review by vordimous — APPROVED** [review]

**Review by jfallows — APPROVED** [review]
